ana-agudelo2/tarea-2/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Ana Agudelo - Tarea 2</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width" />
+        <link rel="stylesheet" href="index.css"/>
+    </head> 
+    
+    <body>
+        <main>
+            <section class="grid-container">
+                <div class="item1">August 2022</div>
```

Comment:
@ana19051 , avoid the use of divs, all these under grid-container should ul within li tags

Reply:
@emolinaab I changed the mistakes 👍